insta/insta.js

```diff
@@ -9,4 +9,13 @@ function init() {
     text.removeClass("collapse");
     text.dotdotdot({}).data("dotdotdot").restore();
   });
-}
\ No newline at end of file
+  $("#humburger").click(function(event) {
+    var humburger = document.getElementById("dropdown");
```

Comment:
document.querySelector("#dropdown") is more modern.  This allows you to have consistent code when selecting either by id "#dropdown" or by class ".thisClass" or by tag type "img", etc.

Reply:
👍